### PR TITLE
Refactor test data as a class

### PR DESCRIPTION
**Pull request by tpmcgowan**
Refactor existing test data to use a class so it can be brought into tests as a single import, with everything then available. Also moved to `./server/routes/testutils` so it's alongside the related existing test setup `appSetup.ts`.

A lot of changes in this PR but most are just updating imports!

### CODE REVIEW

**Review by hutcheonb-moj — APPROVED** [review]
LGTM

File: server/routes/testutils/testData.ts [hunk 1]
```diff
@@ -0,0 +1,216 @@
+import { Prison } from '../../@types/bapv'
+import { SessionCapacity, SupportType, Visit } from '../../data/visitSchedulerApiTypes'
+import { InmateDetail, CaseLoad, PrisonerBookingSummary } from '../../data/prisonApiTypes'
+import { CurrentIncentive, Prisoner } from '../../data/prisonerOffenderSearchTypes'
+
+export default class TestData {
```
Comment (psoleckimoj) [inline server/routes/testutils/testData.ts:6]: Do we want to use `static` methods? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static

Comment (tpmcgowan) [reply server/routes/testutils/testData.ts:6]: Yes!

**Review by psoleckimoj — APPROVED** [review]
LGTM!

**Pull request merged**